gtda/images/preprocessing.py

```diff
@@ -201,3 +201,115 @@ def transform(self, X, y=None):
         Xt = np.concatenate(Xt)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class ImageToPointCloud(BaseEstimator, TransformerMixin):
+    """Transformer returning a collection of point clouds corresponding to the
```

Comment:
My original suggestion for this one-liner wasn't great but the current is a little too long I feel. How about:

```
"""Represent active pixels in 2D/3D binary images as points in 2D/3D space
with coordinates given by their array indices.
```

Reply:
I have made it shorter, and add a description of a second paragraph.